_data/eduantvarmau/eduantvarmau.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Eduardo Antonio Vargas Mauricio
```

Comment:
```suggestion
name: Eduardo Antonio Vargas
```

Reply:
@eduantvarmau mind committing this change clicking on the commit suggestion button above and requesting another review?

Reply:
I can't approve this change because my second name is being removed :c

Reply:
We'll need to reduce it to make it fit the required length